src/routers/form.router.ts

```diff
@@ -1,3 +1,70 @@
-import { Router } from "express";
+import { Router, json } from "express";
+import { HttpError } from "../utils/errors";
+
+import { createAndAddResponseToForm, createForm, getAllForms, getFormResponse } from "../controllers/form.controller";
 
 export const formRouter = Router();
+formRouter.use(json());
+
+//POST new form
+formRouter.post("/", async (req, res) => {
+    try {
+        const newForm = await createForm(req.body);
+        res.status(201).json(newForm);
+    } catch (err: unknown) {
+        if (err instanceof HttpError) {
+            res.status(err.errorCode).json({ error: err.message });
+        } else {
+            res.status(500).json({ error: "An unknown error occurred while creating form" });
+        }
+    }
+})
+
+//GET all forms
+formRouter.get("/", async (req, res) => {
+    try {
+        const allForms = await getAllForms();
+        res.status(200).json(allForms);
+    }
+    catch (err: unknown) {
+        if (err instanceof HttpError) {
+            res.status(err.errorCode).json({ error: err.message });
+        } else {
+            res.status(500).json({ error: "An unknown error occurred while getting all forms" });
+        }
+    }
+})
+
+//PUT new response
+formRouter.put("/forms/:formID", async (req, res) => {
+    try {
+        const form = await createAndAddResponseToForm(
+            req.params.formID,
+            req.body.responseFields
```

Comment:
this can just be req.body since we are only sending the response information in the body object